internal/scorecard/alpha/scorecardutil/bundle.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scorecardutil
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/operator-framework/operator-registry/pkg/registry"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/yaml"
+)
+
+// TODO(joelanford): This function should probably be upstreamed into
+//     operator-framework/operator-registry and improved. Much of the code
+//     upstream already exists, but not in a consumable way. See:
+//     https://github.com/operator-framework/operator-registry/blob/2bdcd07e6f9ea23a35f1fdbb6a7c6584ec53a0a1/pkg/registry/imageinput.go#L25-L81
+//
+// LoadBundleDirectory parses a Bundle from a given on-disk path.
+func LoadBundleDirectory(bundlePath string) (*registry.Bundle, error) {
+	metadataDir := filepath.Join(bundlePath, "metadata")
+	manifestsDir := filepath.Join(bundlePath, "manifests")
+
+	annotationsPath := filepath.Join(metadataDir, "annotations.yaml")
+	annotations, err := decodeAnnotationsFile(annotationsPath)
+	if err != nil {
+		return nil, err
+	}
+
+	files, err := ioutil.ReadDir(manifestsDir)
+	if err != nil {
+		return nil, err
+	}
+
+	fileStrings := []string{}
+	for _, f := range files {
+		if f.IsDir() {
+			continue
+		}
+
+		fStr, err := ioutil.ReadFile(filepath.Join(manifestsDir, f.Name()))
+		if err != nil {
+			return nil, err
+		}
+		fileStrings = append(fileStrings, string(fStr))
+	}
+	return registry.NewBundleFromStrings(annotations.GetName(), annotations.GetName(), annotations.GetDefaultChannelName(), fileStrings)
+}
+
+func decodeAnnotationsFile(path string) (*registry.AnnotationsFile, error) {
+	annotationsData, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+	annotations := registry.AnnotationsFile{}
+	if err := yaml.Unmarshal(annotationsData, &annotations); err != nil {
+		return nil, err
+	}
+	return &annotations, nil
+}
+
+// TODO(joelanford): bump to Kubernetes v1.18 and latest operator-registry
+//  to get access to dependencies file definitions.
+//func decodeDependenciesFile(path string) (*registry.DependenciesFile, error) {
```

Comment:
This is a relatively new item available in `metadata/`. Do does the scorecard need to "understand" dependencies just yet?